src/lib/mina_base/account.ml

```diff
@@ -1036,3 +1036,67 @@ let deriver obj =
        ~permissions:!.Permissions.deriver
        ~zkapp:!.(option ~js_type:Or_undefined (Zkapp_account.deriver @@ o ()))
        obj
+
+module Unstable = struct
```

Comment:
I created this type to simulate the new account type. I didn't want to mess with the versioning types because I was getting indications that it wasn't a `V3`, but more like a `Post_hardfork`.

The only difference is that the `nonce` field is duplicated into a field named `unstable_field`